main.py

```diff
@@ -8,12 +9,13 @@ def main():
     """
     year = "2021"
     cve_year_collection = CVEsCollection(year)
-    cve_dict = cve_year_collection.getCVE_Items()
-    cve_year_collection.getCveByIndex(0)
+    cve = cve_year_collection.getCveByIndex(51)
     print("Hello")
```

Comment:
CVEsCollection: why do u save the number of cves (instead of just return len(cves)) or something?